cacerts/certs_test.go

```diff
@@ -186,6 +186,41 @@ func testCerts(t *testing.T, context spec.G, it spec.S) {
 			Expect(cacerts.CanonicalString("SOME VAL")).To(Equal("some val"))
 		})
 	})
+
+	context("SplitCerts", func() {
+		var dir string
+		it.Before(func() {
+			var err error
+			dir, err = ioutil.TempDir("", "multi-certs")
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		it.After(func() {
+			Expect(os.RemoveAll(dir)).To(Succeed())
+		})
+
+		it("splits file with X certs into X new files", func() {
+			paths, err := cacerts.SplitCerts(filepath.Join("testdata", "multiple-certs.pem"), dir)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(paths).To(HaveLen(2))
+			Expect(paths).To(ConsistOf(
+				ContainSubstring("extra_0_multiple-certs.pem"),
+				ContainSubstring("extra_1_multiple-certs.pem"),
+			))
+		})
+		it("splits file with 1 cert into 1 new file", func() {
+			paths, err := cacerts.SplitCerts(filepath.Join("testdata", "SecureTrust_CA.pem"), dir)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(paths).To(HaveLen(1))
+			Expect(paths).To(ConsistOf(
+				ContainSubstring("extra_0_SecureTrust_CA.pem"),
+			))
+		})
```

Comment:
Not really a problem with the test, but perhaps easier to explain here. Can we make it so that when there is only one cert, it doesn't split and rename the file? If I pass in `SecureTrust_CA.pem` as my CA cert file, it's kind of unexpected to have that transformed into a different file name.

It's not a huge functional difference, but I could possibly see that being confusing. Why did we process a file that didn't need to be processed?

Warning half-baked idea ahead, but I think if you check `rest` on the first time through the loop and it's not empty, then we know it's a multiple CA cert bundle. One way to do that off the top of my head would be to rewrite the loop as a `for <condition> {` loop or just use `for {` and a break inside the loop. Then you can call `pem.Decode()` once before you enter the loop, check rest and set a bool like isMultiple which you can reference in the loop to determine the file name.